javascript/package.json

```diff
@@ -19,7 +19,9 @@
   "files": [
     "dist/binaries/**",
     "dist/src/**",
-    "src/**"
+    "src/**",
+    "load.js",
+    "load.js.map"
```

Comment:
I think this is missing `load.d.ts`

Reply:
Missed a commit, thanks for spotting this.